docs/diagnostics/PX1099.md

```diff
@@ -92,14 +92,14 @@ The following table shows the default banned APIs that are included in the Acumi
-| `System.Data.Entity`                               | Namespace |      ISV     | Usage of this namespace is forbidden in Acumatica customizations.             |
-| `System.Data.SqlClient`                            | Namespace |      ISV     | Usage of this namespace is forbidden in Acumatica customizations.             |
-| `System.AppDomain`                                 |   Type    |      ISV     | Usage of the AppDomain type is forbidden in Acumatica customizations.             |
-| `System.Diagnostics.ProcessStartInfo`              |   Type    |      ISV     | Usage of the ProcessStartInfo type is forbidden in Acumatica customizations.      |
-| `System.Environment`                               |   Type    |      ISV     | Usage of the Environment type is forbidden in Acumatica customizations.           |
-| `System.IO.File`                                   |   Type    |      ISV     | Usage of the File type is forbidden in Acumatica customizations.                  |
+| `System.Data.Entity`                               | Namespace |      ISV     | Usage of this namespace is forbidden in Acumatica customization projects.             |
+| `System.Data.SqlClient`                            | Namespace |      ISV     | Usage of this namespace is forbidden in Acumatica customization projects.             |
+| `System.AppDomain`                                 |   Type    |      ISV     | Usage of the AppDomain type is forbidden in Acumatica customization projects.             |
+| `System.Diagnostics.ProcessStartInfo`              |   Type    |      ISV     | Usage of the ProcessStartInfo type is forbidden in Acumatica customization projects.      |
+| `System.Environment`                               |   Type    |      ISV     | Usage of the Environment type is forbidden in Acumatica customization projects.           |
+| `System.IO.File`                                   |   Type    |      ISV     | Usage of the File type is forbidden in Acumatica customization projects.                  |
 | `System.GC`                                        |   Type    |      ISV     | Access to garbage collector is forbidden in Acumatica customizations. |
-| `System.OperatingSystem`                           |   Type    |      ISV     | Usage of the OperatingSystem type is forbidden in Acumatica customizations.       |
+| `System.OperatingSystem`                           |   Type    |      ISV     | Usage of the OperatingSystem type is forbidden in Acumatica customization projects.       |
 | `System.Reflection.MethodInfo`                     |   Type    |      ISV     | Usage of reflection is forbidden in Acumatica customizations.            |
```

Comment:
@EAndrosova. judging from the fixes made to other table rows, shouldn't you also replace `customizations` with `customization projects` here for `MethodInfo`, `WebConfig`, and `GC`?